ush/chgres_cold2fv3.py

```diff
@@ -6,6 +9,13 @@
 import sys
 
 def nan_check(arr, name, check_id):
+    """
+    NAN check?
+
+    arr: ???
+    name: ???
+    check_id: ???
+    """
```

Comment:
@delippi what is this function for, and what does each argument mean? What does it return?

Reply:
"""
    NAN check function.
    
    This function checks a given array for NaN (Not a Number) values, counts the total number of NaNs, 
    and prints their indices if any are found. It helps identify problematic data in the array that may 
    disrupt further processing.
    
    Parameters:
    arr: numpy array to be checked for NaN values.
    name: string used to label the array in the output for identification.
    check_id: integer or string identifier to differentiate between multiple checks in the output.
    
    Returns:
    nan_count: integer representing the total number of NaN values found in the array.
    """

After all nan_checks are done, if there is at least one nan then the code fails. I put a note right after the vertical remapping of the wind that says "Perform a NaN check - sometimes will get NaNs at this point". I don't think it should fail. I think it was more of an issue in development and I kept the check there just to be safe.